Podcast/BookmarkTableViewCell.swift

```diff
@@ -171,7 +171,7 @@ class BookmarkTableViewCell: UITableViewCell {
     }
     
     func setPlayButtonToState(isPlaying: Bool) {
-        playButton.isEnabled = !isPlaying
+        playButton.isSelected = isPlaying
```

Comment:
Wait I made this disabled while playing on purpose because there was a bug when you held it down.